main.go

```diff
@@ -13,30 +13,31 @@ import (
 )
 
 var (
-	varsFiles     FlagSlice
-	secretFiles   FlagSlice
-	secretName    string
-	configMapName string
-	namespace     string
-	convertKeys   bool
-	flagSet       *flag.FlagSet
+	varsFiles      FlagSlice
+	secretFiles    FlagSlice
+	configMapFiles FlagSlice
+	name           string
+	namespace      string
+	convertKeys    bool
+	flagSet        *flag.FlagSet
 )
 
 func init() {
 	// workaround to avoid inheriting vendor flags
 	flagSet = flag.NewFlagSet("kenv", flag.ExitOnError)
-	flagSet.StringVar(&configMapName, "config-map", "", "Name to give the ConfigMap")
-	flagSet.StringVar(&secretName, "secret-name", "", "Name to give the Secret resource")
+	flagSet.StringVar(&name, "name", "", "Name to give the ConfigMap and Secret resources")
 	flagSet.StringVar(&namespace, "namespace", "default", "Namespace to create the ConfigMap in")
 	flagSet.BoolVar(&convertKeys, "convert-keys", false, "Convert ConfigMap keys to support k8s version < 1.4")
-	flagSet.Var(&varsFiles, "v", "File containing environment variables (repeatable)")
-	flagSet.Var(&secretFiles, "s", "File containing secret environment variables (repeatable)")
+	flagSet.Var(&varsFiles, "v", "Files containing variables to inject as environment variables (repeatable)")
+	flagSet.Var(&secretFiles, "s", "Files containing variables to inject as Secrets (repeatable)")
+	flagSet.Var(&configMapFiles, "c", "Files containing variables to inject as ConfigMaps (repeatable)")
```

Comment:
So rather than take all vars in via `-v`, `-c` will create config maps, `-s` will create secrets, and `-v` will create normal env vars?  Maybe we can add a few examples showing this to the repo.